django/db/backends/postgresql/operations.py

```diff
@@ -265,9 +277,7 @@ def sequence_reset_sql(self, style, model_list):
                             style.SQL_TABLE(qn(model._meta.db_table)),
                         )
                     )
-                    # Only one AutoField is allowed per model, so don't bother
-                    # continuing.
-                    break
```

Comment:
Is there a check for two `AutoField`s? If not perhaps we should add one, though perhaps in a separate ticket/PR.

Reply:
Currently, `AutoField`s are always primary keys, so it's impossible to have multiple auto fields. I re-opened the ticket to revisit this topic recently [#8576](https://code.djangoproject.com/ticket/8576).